article-analysis-project/article-processing.py

```diff
@@ -0,0 +1,207 @@
+from __future__ import print_function
+from newspaper import Article
+from genderize import Genderize
+import nltk
+from nltk.tokenize import sent_tokenize
+import gender_guesser.detector as gender
+import spacy 
+import numpy as np
+import json
+import os
+import itertools
+import string
+import urllib
+import requests, mediacloud.api
+from collect_articles import article_urls
+from bs4 import BeautifulSoup
+import re
+import pandas as pd
+def main():
+    urls = article_urls(25499)
+    
+    complete = []
+    for URL in urls:
+        #URL = 'https://abcnews.go.com/Politics/white-house-odd-couple-trump-boltons-tumultuous-relationship/story?id=71323127'
+        article = Article(URL)
+        article.download()
+        article.parse()
+        main_body = article.text #Main body of Article
```

Comment:
This kind of comment is redundant, since I could look at the names of the variables and realize that main_body contains the text of an article.

The point is: kudos for good variable names that create self documenting code!